lib/application/logic/speaker_cubit.dart

```diff
@@ -0,0 +1,33 @@
+import 'dart:math';
+
+import 'package:deep_conference/application/logic/speaker_state.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import '../../domain/repositories/speaker_repository.dart';
+
+class SpeakerCubit extends Cubit<SpeakerState> {
+  SpeakerCubit(this.speakerRepository, this.speakerIds)
+      : super(const SpeakerState(speakers: [])) {
+    if (speakerIds.first != "") {
+      readSpeakerItems(speakerIds);
+    } else {
+      emit(const SpeakerState(speakers: [], loading: false));
+    }
+  }
+
+  final SpeakerRepository speakerRepository;
+  final List<String> speakerIds;
+
+  void readSpeakerItems(List<String> speakerIds) async {
+    try {
+      final data = await speakerRepository.getSpeakerList(speakerIds);
+      if (data.isNotEmpty) {
+        emit(SpeakerState(speakers: data, loading: false));
+      } else {
+        emit(const SpeakerState(speakers: [], loading: false));
+      }
+    } catch (e) {
+      print(Exception(e));
```

Comment:
Možeš maknuti ovaj print, ako baš želiš printati nešto onda koristi log metodu i možeš direktno predati `e` iz catch-a. Nepotrebno je koristiti Exception, kada je vrlo vjerojatno `e` Exception sam po sebi